src/datasources/blockchain/blockchain-api.manager.ts

```diff
@@ -59,6 +59,7 @@ export class BlockchainApiManager implements IBlockchainApiManager {
     };
   }
 
+  // Note: this assumes Infura as provider when using an API key as authentication method.
```

Comment:
Let me know if you think this note is redundant @iamacook, but I think we have to be aware that if a `Chain` is configured with a different RPC provider, this function will try to use `INFURA_API_KEY` with the other provider URL, right?

Reply:
It will but I think it's fine. It's the same implementation as the Web.

We could alternatively check that the value contains `"infura"` in it. What do you think?

Reply:
I'm totally OK with checking the URL, will do it separately 👍🏻